### PR TITLE
Fix content in user attributes.

### PR DESCRIPTION
**Pull request by ChamathNS**
## Purpose
Fixes https://github.com/wso2-enterprise/asgardeo-product/issues/1537.

Addressed points [1] and [3] as follows,
Screenshot(6)

Addressed point [2] as follows,
Screenshot(7)

Point [4] would be obsolete with the fix for https://github.com/wso2-enterprise/asgardeo-product/issues/1535.

### CODE REVIEW

**Comment by malithie** [comment]
IMO, the description regarding mandatory should go to the tool tip of "Mandatory"
@brionmario , @jeradrutnam thoughts ?

**Comment by jeradrutnam** [comment]
Title description should not have the info message style

File: apps/console/src/features/applications/components/settings/attribute-management/attribute-selection.tsx [hunk 1]
```diff
@@ -468,6 +471,13 @@ export const AttributeSelection: FunctionComponent<AttributeSelectionPropsInterf
                                                     )
                                                 }
                                             </Table.Row>
+                                            <Table.Row>
+                                                <Table.Cell colSpan='2'>
+                                                    <Hint>
+                                                        { t("console:develop.features.applications.edit.sections.attributes.selection.mandatoryAttributeHint") }
```
Comment (DimalChandrasiri) [inline apps/console/src/features/applications/components/settings/attribute-management/attribute-selection.tsx:477]: Possible line length exceeding. Stick to under 120 characters per line. Check other places as well.

**Review by DimalChandrasiri — APPROVED** [review]

**Comment by DimalChandrasiri** [comment]
> Title description should not have the info message style.
> 
> @malithie yes, explaining mandatory checkbox should go in to a tooltip with a help icon. @DimalChandrasiri already did it sometime back. Please check with him as well.

The implementation was added to the mapped user attribute column to explain what it is. The implementation can be found [here.](https://github.com/wso2/identity-apps/blob/a574e248815746bfe1b633f3c12a6f2cf5bc0078/apps/console/src/features/applications/components/settings/attribute-management/attribute-selection.tsx#L519)

Screenshot 2021-02-08 at 14 46 36

**Comment by ChamathNS** [comment]
With https://github.com/wso2/identity-apps/pull/1667/commits/51e35a9bc39f3510f2d73dcf1a7de391438dfed7, added the tool tip for mandatory header. 

attribute_content

File: apps/console/src/features/applications/components/settings/attribute-management/attribute-selection.tsx (hunk 1 above)
Comment (brionmario) [reply apps/console/src/features/applications/components/settings/attribute-management/attribute-selection.tsx]: Fixed with `65ed0fc`

**Pull request merged**